docs/calico-with-docker/AWS.md

```diff
@@ -173,13 +173,15 @@ aws ec2 run-instances \
   --key-name mykey \
   --security-group-ids $SECURITY_GROUP_ID \
   --user-data file://<PATH_TO_CLOUD_CONFIG>/user-data-first
-#  --subnet $SUBNET_ID 
+#  --subnet $SUBNET_ID
 #  Include the subnet param above if using a non-default VPC
+
+INSTANCE_ID_1=<InstanceId>
```

Comment:
We've generally tried to keep the large commands as copy-and-pasteable as possible - this makes it less frustrating for people who are trying to follow the guide.

e.g:
- copy the instance ID and export it, copy this command and execute it.

rather than:
- Copy this command somewhere.  Copy the instance ID somewhere. Delete the bit that was a template, fill it in with the instance ID.  Copy and paste that into the terminal (or similar steps).

Especially when you need to capture a bit of configuration in one part of the guide, and then use it again later on, environment variables help with that.  We've received feedback from various docs testers, including @lwr20 that the environment variable method is a better UX.